go.mod

```diff
@@ -2,6 +2,9 @@ module github.com/upbound/upjet
 
 go 1.19
```

Comment:
Since this is a substantial update to Upjet I wonder if if now is a good time to go up to go 1.20 as well

Reply:
Sure i can add that

Reply:
In general, I would prefer doing such a version upgrade in a separate/dedicated PR not to mix things up.

Reply:
Makes sense to do it in a separate PR :+1:

Reply:
Alper implemented the upgrade to 1.20 here: https://github.com/upbound/upjet/pull/236

Reply:
Awesome, thanks @ulucinar !